lib/stdlib/test/timer_simple_SUITE.erl

```diff
@@ -89,11 +194,24 @@ init_per_testcase(_, Config) when is_list(Config) ->
 
 %% Testing timer interface!!
 
-%% Test of apply_after, with sending of message.
-apply_after(Config) when is_list(Config) ->
+%% Test of apply_after with time = 0, with sending of message.
+apply_after1(Config) when is_list(Config) ->
+    timer:apply_after(0, ?MODULE, send, [self(), ok_apply]),
```

Comment:
Please test all return values to ensure that they follow the documentation.

Reply:
Ok, will do :)

Reply:
Done in `timer_SUITE` and `timer_simple_SUITE`.